.changelog/17066.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+command: Allows creating ACL Token TTL with greater than 24 hours with the -expires-ttl flag.
```

Comment:
nit: per the style guide in the eng doc we should be using the imperitive for changelog entries for consistency.
```suggestion
command: Allow creating ACL Token TTL with greater than 24 hours with the -expires-ttl flag.
```